faststream/nats/publisher/producer.py

```diff
@@ -134,6 +134,7 @@ async def publish(  # type: ignore[override]
         stream: Optional[str] = None,
         timeout: Optional[float] = None,
         rpc: bool = False,
+        rpc_prefix: str = "",
```

Comment:
Can we do not add an extra parameter, but reuse this nats-py code: https://github.com/nats-io/nats.py/blob/main/nats/aio/client.py#L1041 ?

Looks, like user should setup this option via `NatsBroker(inbox_prefix=...)` now and I want to save this behavior

Reply:
I did it this way because your original implementation doesn't use nats inbox creation, so to keep true to that I didn't want to change it up. when you use inbox the reply_to will be some form of _inbox.<id> when you use the broker inbox prefix, which can be passed in if I am not mistaken in a few spots, you get <prefix>_inbox.<id>, while your reply to uses only an id, which doesn't have enough consistency for nats permissions setup. To make this functionality optional is why I used rpc_prefix. Hopefully that explains, otherwise you will want to use nats features all the way through ( in essence I am making an rpc prefix and not broker inbox prefix as they don't necessarily work the same)

Reply:
To add more info, I would replace the rpc id generation with with " nats.NewInbox() "